### PR TITLE
drop Android 4.3, new min version is Android 4.4

### PR DESCRIPTION
**Pull request by AndyScherzinger**
As per #6746, drop Android 4.3 support

### Testing 
Writing tests is very important. Please try to write some tests for your PR. 
If you need help, please do not hesitate to ask in this PR for help.

[unit tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#unit-tests)
[instrumented tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#instrumented-tests)
[UI tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#ui-tests)

- [x] Tests written, or not not needed

### CODE REVIEW

File: src/main/java/com/owncloud/android/utils/FileStorageUtils.java [hunk 1]
```diff
@@ -652,7 +650,7 @@ private static boolean canListFiles(File f) {
         public static final StandardDirectory DOCUMENTS;
 
         static {
-            if (SDK_INT > Build.VERSION_CODES.KITKAT) {
+            if (SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```
Comment (tobiasKaminsky) [inline src/main/java/com/owncloud/android/utils/FileStorageUtils.java:653]: Is this needed?

Comment (tobiasKaminsky) [reply src/main/java/com/owncloud/android/utils/FileStorageUtils.java:653]: I think we can remove this, as AS states that DIRECTORY_DOCUMENTS requires API19.

Comment (AndyScherzinger) [reply src/main/java/com/owncloud/android/utils/FileStorageUtils.java:653]: Nope, the statements said >KITKAT, so basically "LOLLIPOP and above", I just rewrote it so we don't have any KITKAT references in the code anymore.

Comment (AndyScherzinger) [reply src/main/java/com/owncloud/android/utils/FileStorageUtils.java]: As discussed, removed since the check was wrong bc it is a v19 feature...

**Review by tobiasKaminsky — APPROVED** [review]

**Pull request merged**